agent/config/config.go

```diff
@@ -58,6 +58,14 @@ func Parse(data string, format string) (c Config, md mapstructure.Metadata, err
 	return c, md, nil
 }
 
+// CacheConfig Cache tunning configuration
+type CacheConfig struct {
+	// EntryFetchMaxBurst max burst size of RateLimit for a single cache entry
+	EntryFetchMaxBurst *int `json:"rate_limit_burst,omitempty" hcl:"rate_limit_burst" mapstructure:"rate_limit_burst"`
```

Comment:
Similar to above, I think we can remove burst, because it should always be 1 (unless I have misunderstood something).